nextflow_schema.json

```diff
@@ -142,40 +154,44 @@
                     "fa_icon": "far fa-file-code",
                     "description": "Run CTAT-splicing to detect abberant cancer splicing introns. Needs --arriba and/or --starfusion to run."
                 },
-                "gencode_ref": {
-                    "type": "string",
-                    "fa_icon": "far fa-file-code",
-                    "description": "Path to gencode references"
-                },
```

Comment:
should not be deleted

Reply:
Why not? It wasn't used in the pipeline :)

Reply:
You are correct. I had a legacy concept in my head. Go ahead!